cmd/stackdriver-reverse-proxy/main.go

```diff
@@ -53,6 +63,8 @@ Options:
 
 Tracing options:
   -trace-fraction Tracing sampling fraction, between 0 and 1.0.
+  -monitor-http	  Turning on reporting HTTP stats to StackDriver Monitoring
```

Comment:
I was wondering whether we should make all of the features on by default. Then, provide flags to turn them off if they don't want to pay/enable the API. What do you think?

Reply:
Great thought right there, I agree we should have this on by default. Thanks for the advocacy to get it on.